css/types/variable.json

```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "variables": {
+        "__compat": {
+          "description": "<code>&lt;variables&gt;</code>",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables",
```

Comment:
This should be: <code>https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables</code>

Reply:
👍